Svc/FileManager/Commands.fppi

```diff
@@ -20,6 +20,7 @@ async command RemoveDirectory(
 @ Remove a file
 async command RemoveFile(
                           fileName: string size 256 @< The file to remove
+                          ignoreErrors: bool @< Ignore non-existent files
```

Comment:
A suggestion here is to make this an enumeration.  That way it is a little more understandable in sequences.  E.g. a sequence:

```
fm.REMOVE_FILE /path/to/file TRUE
```
vs 
```
fm.REMOVE_FILE /path/to/file IGNORE_ERRORS
```
Using an enum self-describes without needing to look-up the command definition to remember which polarity the bool represented.

Reply:
I agree with this approach. This one has gotten a little stale. @codeflight1: Are you willing to update this PR?

Reply:
I definitely can, but I was waiting for a decision on whether that should be added to each command or made a component-wide parameter (see other requested changes).